apgar-probe.go

```diff
@@ -155,6 +156,7 @@ func main() {
 	flag.StringVar(&documentRoot, "document-root", "/var/lib/apgar", "Document root")
 	flag.StringVar(&healthCheckName, "healthcheck-name", "healthCheck", "health check script suffix")
 	flag.StringVar(&healthCheckTree, "healthcheck-tree", "/var/lib/apgar", "Directory tree to search for health checks")
+	flag.StringVar(&healthyResponse, "healthy-response", "200 OK\n", "The response string if status is healthy")
```

Comment:
Do you think we need to new line here? aws document just said "200 OK", not sure if new line will affect.